test/testQueryClient.ts

```diff
@@ -44,5 +49,34 @@ export const queryClientWithMockData = (): QueryClient => {
     () => LEDERE_DEFAULT
   );
 
+  // TODO Forbedre/flytt dette til et eget sted. Fjerne?
```

Comment:
Fix TODO? Kan muligens bruke daysFromToday i testUtils.ts til dette (samme gjelder for testData.ts over)